### PR TITLE
Initial commit of Top Level Search page

### PR DESCRIPTION
**Pull request by elarens**
Created a draft Top Level Search page

### CODE REVIEW

**Review by nataliemona — APPROVED** [review]

File: frontend/src/components/organisms/TopLevelSearchFormFields/TopLevelSearchFormFields.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    color: whitesmoke !important;
+    background-color: rgba(27, 39, 53, 0.95) !important;
+}
+
+.ant-collapse > .ant-collapse-item > .ant-collapse-header {
+    color: white !important;
+    background-color: rgba(27, 39, 53, 0.85) !important;
+}
+
+/* .ant-radio-button-wrapper {
```
Comment (nataliemona) [inline frontend/src/components/organisms/TopLevelSearchFormFields/TopLevelSearchFormFields.css:59]: Best not to leave commented out code in.

File: frontend/src/components/organisms/TopLevelSearchFormFields/TopLevelSearchFormFields.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        name="country"
+        label="Country"
+        tooltip={{
+          title: "The 'Country' field is used to find clinical studies with locations in a specific country. For example, if you choose the United States, you can then narrow your search by selecting a state and identifying a city and distance.",
+          icon: <InfoCircleOutlined />,
+        }}
+      >
+        <Select
+          placeholder="Select the country"
+            // onChange={this.onGenderChange}
```
Comment (nataliemona) [inline frontend/src/components/organisms/TopLevelSearchFormFields/TopLevelSearchFormFields.js:118]: Commented out code.

**Comment by bapower** [comment]
In TopLevelSearchFormFields, each item in the list of form items should have a unique key

**Comment by bapower** [comment]
remove commented out code.
For example:
Screen Shot 2021-03-17 at 00 07 12

Screen Shot 2021-03-17 at 00 15 53

Screen Shot 2021-03-17 at 00 47 36

**Comment by bapower** [comment]
The form element should be in TopLevelSearchForm, not in TopLevelSearchFormFields

**Comment by bapower** [comment]
TopLevelSearchForm should probably be an organism and TopLevelSearchFields should probably be a molecule as it is a smaller part of the TopLevelSearchForm

**Comment by bapower** [comment]
The views should be in the components folder as they are components

**Comment by bapower** [comment]
What is the logic for whether something is a functional or class component? A few components that you have as classes do not have state, which I thought was the rationale for using functional components. There should be some consistency.

**Comment by bapower** [comment]
Rather than hard code all of the form items, I think it would make sense to manage them as props. That way if something needs to be updated there is one place to go to update it and it keeps the render code cleaner.

**Comment by bapower** [comment]
Avoid using !important in css as it messes with the cascade and can make debugging the css difficult. Instead use a selector with greater specificity.

**Comment by bapower** [comment]
the form inputs should be managed as state in order to do any form handling and to take advantage of the data binding that react offers

**Comment by bapower** [comment]
If you use hooks, it makes it impossible to convert the component to a class component in the future without major refactoring

**Comment by bapower** [comment]
You should make button and input field into components so they can be re-used.

**Pull request merged**